spec.bs

```diff
@@ -1222,6 +1222,78 @@ Each {{FencedFrameConfig}} has:
   1. Initialize |value|'s [=fencedframeconfig/contentHeight=] to |serialized|.\[[ContentHeight]].
 </div>
 
+To help with ease of adoption,
+[until 2026](https://github.com/WICG/turtledove/issues/286#issuecomment-1682842636) we will support
```

Comment:
This makes it sound like we're actually indeed removing the API in 2026. Are we? Or are we just "deprecating" it softly then?

Reply:
It has been deprecated since it was added, hence the name. It should actually be removed in several years.